src/main/kotlin/com/exactpro/th2/codec/api/IPipelineCodecContext.kt

```diff
@@ -19,6 +19,9 @@ package com.exactpro.th2.codec.api
 import com.exactpro.th2.common.schema.dictionary.DictionaryType
 import java.io.InputStream
 
-fun interface IPipelineCodecContext {
-    operator fun get(type: DictionaryType): InputStream
+interface IPipelineCodecContext {
+    @Deprecated("Dictionary types will be removed in future releases of infra", ReplaceWith("getByAlias(alias)"))
+    fun getByType(type: DictionaryType): InputStream
+    fun getByAlias(alias: String): InputStream
```

Comment:
Method signature doesn't make it clear what we're getting by alias. I'd suggest overloading `get` operator like this:

```operator fun get(dictionaryAlias: String): InputStream```